packages/odyssey-react/src/utils/useKeypress.test.ts

```diff
@@ -0,0 +1,31 @@
+/*!
+ * Copyright (c) 2021-present, Okta, Inc. and/or its affiliates. All rights reserved.
+ * The Okta software accompanied by this notice is provided pursuant to the Apache License, Version 2.0 (the "License.")
+ *
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0.
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *
+ * See the License for the specific language governing permissions and limitations under the License.
+ */
+
+
+import { fireEvent } from '@testing-library/react';
+import { renderHook, act } from '@testing-library/react-hooks'
+import { useKeypress } from "./useKeypress";
+
+test('should fire callback when key in map is pressed', () => {
```

Comment:
can you restructure this test to include a `describe('useKeyPress', () =>` wrapping function so the output is a bit nicer? thank you!